app/models/need.rb

```diff
@@ -87,18 +91,32 @@ def status=(state)
   end
 
   def self.base_query
-    Need.includes(:contact, :user)
+    sql = "LEFT JOIN (select c.id, max(nt.created_at) from contacts c
```

Comment:
Would be good to have test coverage for this.  Not to stop this going through, but we need to make sure we have some rspec or cucumber tests to cover this.